conda_smithy/templates/circle.yml.tmpl

```diff
@@ -11,10 +11,7 @@ general:
 {%- if matrix -%}
 checkout:
   post:
-    # Checkout the merged PR for testing as CircleCI will just use the PR head otherwise.
-    - if [[ -n "${CIRCLE_PR_NUMBER}" ]]; then git fetch origin +refs/pull/$CIRCLE_PR_NUMBER/head:pr/$CIRCLE_PR_NUMBER/head ; fi
-    - if [[ -n "${CIRCLE_PR_NUMBER}" ]]; then git fetch origin +refs/pull/$CIRCLE_PR_NUMBER/merge:pr/$CIRCLE_PR_NUMBER/merge ; fi
-    - if [[ -n "${CIRCLE_PR_NUMBER}" ]]; then git checkout -qf pr/$CIRCLE_PR_NUMBER/merge ; fi
+    - ./ci_support/checkout_code.sh
```

Comment:
I would probably rename this script to something a little more descriptive, but otherwise, I have no comments on the content of the script itself - I'm happy to leave that to your greater experience of Circle in this regard.